lib/indico.js

```diff
@@ -30,9 +31,15 @@ indico.Collection = function(collectionName, config) {
 }
 
 indico.collections = function(config) {
-  return make_request('/custom/collections', {}, false, config);
+  return makeRequest('/custom/collections', {}, false, config);
 }
 
+indico.relevance = function(data, queries, config) {
```

Comment:
Fair enough. If we add more apis like that, we should add an optional `required_params` key to services and handle that in `make_request`.